docs/modules/ROOT/examples/tutorial/minio/deployment.yaml

```diff
@@ -26,6 +26,8 @@ spec:
         args:
         - server
         - /data
+        - '--console-address'
+        - ':9001'
         env:
         - name: MINIO_ROOT_USER
           valueFrom:
```

Comment:
I can't review and suggest directly in this PR, but around line 43 you also need to expose the container port 9001 next to port 9000, otherwise the Service cannot be created.

Reply:
Interesting, it worked without that, I'll add it.